### PR TITLE
add text and tooltip in co2 emissions charts

### PR DESCRIPTION
**Pull request by Esteban100k**
CO2 stats will now have the following description:

The CO2 emissions shown are summarized of both the Camino and the Columbus network. Moreover, if a particular network is selected, the CO2 emissions will be displayed collectively.

### CODE REVIEW

File: src/app/components/Statistics/CO2ConsumptionCharts/index.tsx [hunk 1]
```diff
@@ -42,13 +44,28 @@ const DateRangeContainer = styled.div`
...
 
+interface TextProps {
+    backgroundColor: string
+}
+
+const Text = styled('p')<TextProps>`
+    margin-left: 3rem !important;
+    margin-right: 1rem !important;
+    margin-top: 0.5rem !important;
+    margin-bottom: 0.5rem !important;
```
Comment (Chriserus) [inline src/app/components/Statistics/CO2ConsumptionCharts/index.tsx]: I don't like the use of `!important` here, is it necessary? If not, can you please remove it?

File: src/app/pages/Statistics/index.tsx [hunk 2]
```diff
@@ -56,6 +56,9 @@ const Statistics: FC = () => {
     const theme = useTheme()
     const dark = theme.palette.mode === 'light' ? false : true
 
+    let descriptionDefaultCO2 =
+        'The CO2 emissions shown are summarized of both the Camino and the Columbus network. Moreover, if a particular network is selected, the CO2 emissions will be displayed collectively.'
```
Comment (Chriserus) [inline src/app/pages/Statistics/index.tsx:60]: Was this description confirmed with product? @tabeaarnold Just want to be sure

File: src/app/components/Statistics/CO2ConsumptionCharts/index.tsx (hunk 1 above)
Comment (Esteban100k) [reply src/app/components/Statistics/CO2ConsumptionCharts/index.tsx]: Removed

File: src/app/pages/Statistics/index.tsx (hunk 2 above)
Comment (Esteban100k) [reply src/app/pages/Statistics/index.tsx:60]: You can find the text on the TECH 790

**Review by Ayagoumi — APPROVED** [review]

**Review by Chriserus — APPROVED** [review]

**Review by aeddaqqa — APPROVED** [review]

**Pull request merged**